src/backend/Csrs.Api/Configuration/SerilogExtensions.cs

```diff
@@ -1,44 +1,44 @@
-﻿using Serilog;
+﻿using Csrs.Api.Configuration;
+using Serilog;
 
-namespace Csrs.Api.Configuration
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
Changed namespace so that in Program.cs, we do not require additional using statements and the `AddXXX` method is just available.